frontend/common/src/components/form/MultiSelect/MultiSelect.tsx

```diff
@@ -69,7 +69,13 @@ export const MultiSelect: React.FunctionComponent<MultiSelectProps> = ({
                 }
                 placeholder={placeholder}
                 options={options}
-                styles={{}}
+                aria-label={label}
+                styles={{
+                  placeholder: (provided) => ({
+                    ...provided,
+                    color: `#646464`,
```

Comment:
Having a raw colour hex in here feels a little weird but I doubt this could be handled in hydrogen and if it's not being used anywhere else it doesn't make sense to have it in the shared CSS file. :shrug:

Reply:
Yeah, I agree. I'm not super happy about doing it this way but I reviewed both `react-select` and `hydrogen` and couldn't find a better way. Unfortunately, `react-select` seems to want it passed as a style object and not `className`.

Reply:
My only comment would be that this might be a good place for a comment explaining that a one-off colour was used here for a11y purposes.